### PR TITLE
Dp 39 make dolphin js polymer tests run on sauce labs

### PR DESCRIPTION
**Pull request by kunsingh**
This includes -
https://github.com/canoo/dolphin-platform/issues/314
https://github.com/canoo/dolphin-platform/issues/311
https://github.com/canoo/dolphin-platform/issues/39
https://github.com/canoo/dolphin-platform/issues/167

### CODE REVIEW

**Comment by hendrikebbers** [comment]
@netopyr maybe you can have a quick look at the changes?

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ addons:
     - google-chrome-stable
 
 before_script:
+  - export DISPLAY=:99.0
```
Comment (netopyr) [inline .travis.yml]: I do not know travis, but this build script looks suspicious.

File: .travis.yml [hunk 2]
```diff
@@ -27,6 +30,11 @@ before_script:
 
 script:
   - gulp clean verify build
+#  - gulp build-test
```
Comment (netopyr) [inline .travis.yml]: Is the code that is commented out still needed?

File: wct.conf.js [hunk 3]
```diff
@@ -1,63 +1,22 @@
...
-          platform: 'iOS',
-          version: '8.4',
-          device: 'iPad Simulator',
-          browserName: 'Safari',
-          deviceOrientation: 'landscape'
+    suites: ['test/wct_runner.html'],
+    testTimeout: 5 * 60 * 1000,
+    plugins: {
+        local: {
+            "browsers": ["chrome", "firefox"] //safari not working
```
Comment (netopyr) [inline wct.conf.js]: You cannot do this. If somebody does not have Chrome or Firefox installed, the tests will fail.

File: wct.conf.js [hunk 4]
```diff
@@ -1,63 +1,22 @@
...
+        },
+        sauce: {
+            disabled: true,
```
Comment (netopyr) [inline wct.conf.js]: You are disabling the sauce-plugin? I thought the point of this change is to run tests on SauceLabs.

File: gulpfile.js [hunk 5]
```diff
@@ -55,16 +52,13 @@ gulp.task('build-test', function() {
...
 //add 'test' task when tests are fixed
-gulp.task('verify', ['lint']);
-
+gulp.task('verify', ['lint', 'test']);
 
+gulp.task('test', ['build-test', 'test:local']);
 
 var mainBundler = browserify(assign({}, watchify.args, {
     entries: './src/dolphin-polymer-api.js',
-    standalone: 'dolphin',
```
Comment (netopyr) [inline gulpfile.js]: Is this change intended?

File: .travis.yml (hunk 1 above)
Comment (kunsingh) [inline .travis.yml]: I think this line I added for this - https://docs.travis-ci.com/user/gui-and-headless-browsers/

File: .travis.yml (hunk 2 above)
Comment (kunsingh) [inline .travis.yml]: I will remove this.

File: wct.conf.js (hunk 3 above)
Comment (kunsingh) [inline wct.conf.js]: Yes...Safari is not working locally and on travis.

File: wct.conf.js (hunk 4 above)
Comment (kunsingh) [inline wct.conf.js]: This is disabled for local only. If you run this as 'wct --plugin sauce' then it will ignore disabled flag. See - https://github.com/Polymer/web-component-tester

**Comment by kunsingh** [comment]
I followed this polycast - https://www.youtube.com/watch?v=afy_EEq_4Go
